about.md

```diff
@@ -40,7 +96,8 @@ Unlike a traditional CMS, Pico doesn't require write access on your server to fu
 
 There really isn't that much more to say about it.  The beauty of Pico is in its simplicity.  If you want to create a new page, you make a new file.  That's it.  But that's far from the end of the story.  Despite Pico's simplicity, you'll find it incredibly powerful, and ready to take on any task.
 
-{% include fancyboxtemplate.html, gallery: workflow %}
```

Comment:
That would explain some of the difficulty I'm having.  I had to break the gallery variable out into it's own line to get the page to build, but then the `for` loop wasn't running.

I was using the syntax I found here: https://help.shopify.com/themes/liquid/tags/theme-tags#include

Didn't realize it was different.